com.unity.ml-agents/Runtime/Demonstrations/DemonstrationRecorder.cs

```diff
@@ -32,6 +32,11 @@ public class DemonstrationRecorder : MonoBehaviour
         [Tooltip("Whether or not to record demonstrations.")]
         public bool Record;
 
+        /// <summary>
+        /// Number of steps to record. Editor will stop playing when it hits this threshold.
+        /// </summary>
+        public int numStepsToRecord = 5000;
```

Comment:
Is there a way to specify infinite steps (i.e. current behavior?)

Reply:
no, i figured if you want more than 5k you could just change the NumStepsToRecord. i'm not sure what the typical range is for the number of steps for demos.